apcd-cms/src/apps/exception/templates/exception_submission_form/exception_form_success.html

```diff
@@ -8,6 +8,6 @@ <h1>Request to Submit Exception</h1>
     <p>
         Your exception submission was successful.
     </p>
-    <a class="c-button c-button--primary" href="/submissions/exception">Go to Exception Forms</a>
+    <a class="c-button c-button--primary" href="/submissions/exception">Go Back to Exception Form</a>
```

Comment:
```suggestion
    <a class="c-button c-button--primary" href="/submissions/exception/">Go Back to Exception Form</a>
```
Back button consistency